package-lock.json

```diff
@@ -0,0 +1,12425 @@
+{
```

Comment:
This file shouldn't exist. You probably used npm instead of pnpm at once point. You can delete this.